slsim/Sources/quasars.py

```diff
@@ -27,8 +28,11 @@ def __init__(
         :param kwargs_variability_model: keyword arguments for the variability of
          a source. This is a population argument, not the light curve parameter for
          the individual quasars.
+        :param light_profile: keyword for number of sersic profile to use in source
```

Comment:
is there an error if the light_profile input is not None?

Reply:
There is no error. If light_profile is other than None in case of point source, Lens class does not use it. I have just raised a user warning if light_profile is not None for point source.